script/Ciel/Day 9/09_01_CIEL09_1C.txt

```diff
@@ -168,7 +168,7 @@ As I'd feared, the familiar sight of the infirmary has been disrupted by a kooky
 [sha:8febc66f9de91019ac6dd070f2ad9d2c8101e5cb]{
 -- Page 65, Offset 22300.
 -- 「あらぁ？　坊ちゃんったらここの生徒だったのぅ？
-"What have we here? So this is where the young master goes to school?
+"What have we here? So this is where my little prince goes to school?
```

Comment:
This is Arach not mario but honestly it feels in character

Reply:
lol, why not, then